lib/db.js

```diff
@@ -721,6 +766,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyTokens = function (tokenVerificationId, tokenData) {
+    log.trace({ op: 'DB.verifyToken', tokenVerificationId: tokenVerificationId })
+    return this.pool.post('/tokens/' + tokenVerificationId.toString('hex') + '/verify',
+        unbuffer(
+          tokenData,
+          'inplace'
+        )
+      )
+  }
```

Comment:
I think this might be ok because the fxa-mysql-db throws the error, https://github.com/mozilla/fxa-auth-db-mysql/blob/phil/fxa-83/lib/db/mem.js#L346